modules/tinymce/src/core/main/ts/api/html/Writer.ts

```diff
@@ -3,16 +3,15 @@ import Tools from '../util/Tools';
 import Entities from './Entities';
 
 /**
- * This class is used to write HTML tags out it can be used with the Serializer or the SaxParser.
+ * This class is used to write HTML tags out it can be used with the Serializer.
  *
  * @class tinymce.html.Writer
+ * @version 3.4
  * @example
- * var writer = new tinymce.html.Writer({indent: true});
- * var parser = new tinymce.html.SaxParser(writer).parse('<p><br></p>');
+ * const writer = tinymce.html.Writer({indent: true});
+ * writer.start('node', { attr: 'value });
```

Comment:
```suggestion
 * writer.start('node', {attr: 'value});
```
For consistency

Reply:
It's meant to have spaces for readability... you're right that the old version doesn't have that though.

Reply:
Have fixed the whitespace in the original and a few others I found since I wasn't looking for this originally. I also fixed the `so` typo you mentioned.

Reply:
Thank you